### PR TITLE
handlers for image scan, store results overview in DB

### PR DESCRIPTION
**Pull request by reasonerjt**

### CODE REVIEW

File: make/docker-compose.clair.yml [hunk 1]
```diff
@@ -35,6 +38,9 @@ services:
     restart: always
     depends_on:
       - postgres
+#    environment:
```
Comment (wy65701436) [inline make/docker-compose.clair.yml]: remove the proxy here

File: make/common/templates/clair/config.yaml [hunk 2]
```diff
@@ -16,7 +16,7 @@ clair:
     # Deadline before an API request will respond with a 503
     timeout: 300s
   updater:
-    interval: 0h
+    interval: 2h
```
Comment (wy65701436) [inline make/common/templates/clair/config.yaml]: will we let user to config these?

Comment (reasonerjt) [inline make/common/templates/clair/config.yaml]: IMO we won't

File: make/docker-compose.clair.yml (hunk 1 above)
Comment (reasonerjt) [inline make/docker-compose.clair.yml]: oops, thanks!

**Pull request merged**